src/vendor/perfherder.js

```diff
@@ -39,7 +40,7 @@ const frameworkCache = {};
 const getFramework = async framework => {
   if (frameworkCache[framework] === undefined) {
     frameworkCache[framework] = (async () => {
-      const url = `${TREEHERDER}/api/project/${repository}/performance/signatures/?${toQueryString(
+      const url = `${TREEHERDER}/api/project/${REPO}/performance/signatures/?${toQueryString(
```

Comment:
Is there an important distinction (scope, perhaps?) between `const`s in lower-case and `const`s in UPPER-CASE?

Reply:
It is assumed that uppercase never changes

Reply:
....well, I guess all `const` never change, but all-caps implies application-wide constant.